UI/MVUX/MVUX/MVUX/App.xaml.cs

```diff
@@ -0,0 +1,131 @@
+                .UseEnvironment(Environments.Development)
+#endif
+                .UseLogging(configure: (context, logBuilder) =>
+                {
+                    // Configure log levels for different categories of logging
+                    logBuilder
+                        .SetMinimumLevel(
+                            context.HostingEnvironment.IsDevelopment() ?
+                                LogLevel.Information :
+                                LogLevel.Warning)
+
+                        // Default filters for core Uno Platform namespaces
+                        .CoreLogLevel(LogLevel.Warning);
+
+                    // Uno Platform namespace filter groups
+                    // Uncomment individual methods to see more detailed logging
+                    //// Generic Xaml events
+                    //logBuilder.XamlLogLevel(LogLevel.Debug);
+                    //// Layout specific messages
+                    //logBuilder.XamlLayoutLogLevel(LogLevel.Debug);
+                    //// Storage messages
+                    //logBuilder.StorageLogLevel(LogLevel.Debug);
+                    //// Binding related messages
+                    //logBuilder.XamlBindingLogLevel(LogLevel.Debug);
+                    //// Binder memory references tracking
+                    //logBuilder.BinderMemoryReferenceLogLevel(LogLevel.Debug);
+                    //// DevServer and HotReload related
+                    //logBuilder.HotReloadCoreLogLevel(LogLevel.Information);
+                    //// Debug JS interop
+                    //logBuilder.WebAssemblyLogLevel(LogLevel.Debug);
+
+                }, enableUnoLogging: true)
+                .UseSerilog(consoleLoggingEnabled: true, fileLoggingEnabled: true)
+                .UseConfiguration(configure: configBuilder =>
+                    configBuilder
+                        .EmbeddedSource<App>()
+                        .Section<AppConfig>()
+                )
+                // Enable localization (see appsettings.json for supported languages)
+                .UseLocalization()
+                // Register Json serializers (ISerializer and ISerializer)
+                .UseSerialization((context, services) => services
+                    .AddContentSerializer(context)
+                    .AddJsonTypeInfo(WeatherForecastContext.Default.IImmutableListWeatherForecast))
+                .UseHttp((context, services) => services
+                    // Register HttpClient
+#if DEBUG
+                    // DelegatingHandler will be automatically injected into Refit Client
+                    .AddTransient<DelegatingHandler, DebugHttpHandler>()
+#endif
+                    .AddSingleton<IWeatherCache, WeatherCache>()
+                    .AddRefitClient<IApiClient>(context))
+                .ConfigureServices((context, services) =>
+                {
+                    // TODO: Register your services
+                    //services.AddSingleton<IMyService, MyService>();
```

Comment:
```suggestion
                    services.AddSingleton<IStateService, StateService>();
```

We need to register the `StateService` so that it get injected to the `StateModel`.